app/workers/zendesk_ticket_worker.rb

```diff
@@ -1,11 +1,20 @@
 class ZendeskTicketWorker
   include Sidekiq::Worker
-  
+
   def perform(ticket_options)
     if GDS_ZENDESK_CLIENT.users.suspended?(ticket_options["requester"]["email"])
       $statsd.increment("#{::STATSD_PREFIX}.report_a_problem.submission_from_suspended_user")
     else
-      GDS_ZENDESK_CLIENT.ticket.create!(HashWithIndifferentAccess.new(ticket_options))
+      begin
+        create_ticket(ticket_options)
+      rescue ZendeskAPI::Error::NetworkError => e
+        raise unless e.response.status == 409
```

Comment:
When the response status isn't `409`, this statement raises a `RuntimeError`, rather than re-raise the original network error. I think it'd be better to re-throw the original error, as it contains all the context on it.